components/NavigationBar/src/MDCNavigationBar.h

```diff
@@ -44,6 +44,31 @@ typedef NS_ENUM(NSInteger, MDCNavigationBarTitleAlignment) {
 
 @end
 
+@class MDCNavigationBar;
+
+/**
+  The MDCNavigationBarAccessibilityEnforcer class creates an external object with which to work on
+  an instance of a Material Navigation Bar to activate and esnure accessibility on its title and 
+  buttons.
+
+  ### Dependencies
+
+  Material NavigationBarAccessibilityEnforcer depends on the Navigation Bar material component and
+  MDFTextAccessibility Framework.
+  */
+
+@interface MDCNavigationBarAccessibilityEnforcer : NSObject
+
+- (nonnull instancetype)init;
+
+/**
+  Mutates title text color and navigation items' tint colors based on background color of
+  navigation bar.
+  */
+- (void)enforceFontColorAccessibility:(nonnull MDCNavigationBar *)navBar;
```

Comment:
actually can we just change this to:
- (void)mutate:(nonnull MDCNavigationBar *)navBar;
or
- (void)mutate:(id)navBar;

this way other mutators can have the same api signature.